tests/integration_tests/test_flattening.py

```diff
@@ -0,0 +1,41 @@
+import gym
+import tensorflow as tf
+
+from garage.envs import normalize
+from garage.np.baselines import LinearFeatureBaseline
+from garage.tf.algos import PPO
+from garage.tf.envs import TfEnv
+from garage.tf.experiment import LocalTFRunner
+from garage.tf.policies import CategoricalMLPPolicy
+from tests.fixtures import snapshot_config, TfGraphTestCase
+from tests.fixtures.envs.wrappers import ReshapeObservation
+
+
+class TestResume(TfGraphTestCase):
+
+    def test_ppo_pendulum_flatten_input(self):
```

Comment:
maybe just put this along with all the other PPO tests?

Reply:
sure